File: the99thTime/Programs_v2/FPOSquareRootBenchmark.py
import math
import os
import random
from datetime import datetime

NUM_ITERATIONS = 3600000

currentDatetime = datetime.now()
dateTimeString = currentDatetime.strftime("%Y%m%d_%H%M%S")

outputDir = os.path.abspath("Output") 
os.makedirs(outputDir, exist_ok=True)

filePath = os.path.join(outputDir, f"sqrtResult_{dateTimeString}.txt")

with open(filePath, "w") as writer:
    for i in range(NUM_ITERATIONS):
        a = random.randrange(20000, 40001)
        writer.write(f"{math.sqrt(a)}\n")

print("Done")

File: OldFiles/refrence_ServerFolder_depriciated/Programs/FPOSquareRootBenchmark.java
import java.util.Random;
import java.io.FileWriter;
import java.io.IOException;
import java.io.PrintWriter;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class FPOSquareRootBenchmark {
    private static final int NUM_ITERATIONS = 3600000;
    private static final int MIN_VALUE = 20000;
    private static final int MAX_VALUE = 40000;

    public static void main(String[] args) {
        LocalDateTime currentDateTime = LocalDateTime.now();

        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyyMMdd_HHmmss");
        String dateTimeString = currentDateTime.format(formatter);

        try (PrintWriter writer = new PrintWriter(new FileWriter("Output/sqrtResult_" + dateTimeString + ".txt"))) {
            Random random = new Random();

            for (int i = 0; i < NUM_ITERATIONS; i++) {
                double randomNumber = getRandomNumber(random);
                double squareRoot = Math.sqrt(randomNumber);
                writer.println(squareRoot);
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
        System.exit(1) ;
    }

    private static double getRandomNumber(Random random) {
        return MIN_VALUE + (MAX_VALUE - MIN_VALUE) * random.nextDouble();
    }
}
